api/controllers/UserController.js

```diff
@@ -89,15 +98,15 @@ module.exports = {
           pNum
         });
       }
-      // FIXME: #965
+      // TODO #965
       // Remove everything between here and catch AFTER gamestate is deployed
       const unpopulatedGame = gameId ? await gameService.findGame({ gameId }) : null;
       const populatedGame =
         unpopulatedGame?.status === GameStatus.STARTED
           ? await gameService.populateGame({ gameId })
           : null;
       req.session.loggedIn = true;
```

Comment:
Todo:

1. Move this up with req.session.usr assignment
2. Write issue to remove isLoggedIn

Reply:
#347